Classes/Domain/Validator/EventValidator.php

```diff
@@ -26,10 +25,8 @@ class EventValidator extends AbstractValidator
 {
     /**
      * Checks, if the given time model is empty or its sub-property timeBegin is filled.
-     *
-     * @param mixed $value
      */
-    public function isValid($value)
+    public function isValid(mixed $value): void
     {
         if (!$value instanceof Event) {
             return;
```

Comment:
Are you sure? the validator shouldn't retirm anything

Reply:
This was not my idea. It's a specification by ValidatorInterface. Errors were added by `$this->addError()`. So, yes, it will not return anything